content/source/docs/enterprise/install/installer.html.md

```diff
@@ -201,15 +203,16 @@ RUN update-ca-certificates
 
 #### RHEL 7
 
-Note that in addition to the packages installed by yum, the `daemontools` packaged is required as it provides the `envdir` binary,
-which is used by TFE. If this package is not installed on the alternative worker image, Terraform plan and apply operations will fail.
+Note that in addition to the packages installed by yum, the Python port
+of the `envdir` tool is also installed as it is used by TFE. If this
+tool is not installed on the alternative worker image, Terraform plan
+and apply operations will fail.
```

Comment:
Also - is TFE defined anywhere as an acronym on this page? I noticed that we switch back and forth between writing out Terraform Enterprise and using TFE.  My recommendation is to at the very top, define it at first use like this:  "Terraform Enterprise (TFE)"  and then just use TFE consistently throughout. But we shouldn't use the acronym unless it's defined, even if we think people probably know what it is  :-) 

```suggestion
In addition to the packages that yum installs, Terraform Enterprise also installs the the `envdir` tool Python port. If the `envdir` tool is not installed on the alternative worker image, Terraform plan and apply operations will fail.
```

Reply:
💯 ❌ 💯

Reply:
I'm very much in favour of consistently defining acronyms 👍🏼 

I modified your suggestion to clarify where and why envdir is installed. What do you think?
```suggestion
In addition to the packages that yum installs, the envdir tool Python port is also installed as it is a dependency of the Terraform Build Worker service. If the envdir tool is not installed on the alternative worker image, Terraform runs will fail within Terraform Enterprise.
```

Reply:
Oh I really like it!! my only final nit is - can we make it not passive? What is doing the installing? In an ideal world, I think we'd want to say something like "In addition to the packages that yum installs, [name of thing] installs the envdir tool Python port because it is a dependency of the Terraform Build Worker service."

Reply:
Ah, sorry. How about this?

```suggestion
In addition to the packages that yum installs, curl installs the envdir tool Python port because it is a dependency of the Terraform Build Worker service. If the envdir tool is not installed on the alternative worker image, Terraform runs will fail within Terraform Enterprise.
```

Reply:
love that - merge it!